mongodbatlas/fw_provider.go

```diff
@@ -70,6 +72,20 @@ type tfAssumeRoleModel struct {
 	SourceIdentity    types.String `tfsdk:"source_identity"`
 }
 
+func (m *tfAssumeRoleModel) AttrTypes() map[string]attr.Type {
+	return map[string]attr.Type{
+		"policy_arns":         types.SetType{ElemType: types.StringType},
+		"transitive_tag_keys": types.SetType{ElemType: types.StringType},
+		"tags":                types.MapType{ElemType: types.StringType},
+		"duration":            types.StringType,
+		"external_id":         types.StringType,
+		"policy":              types.StringType,
+		"role_arn":            types.StringType,
+		"session_name":        types.StringType,
+		"source_identity":     types.StringType,
+	}
+}
```

Comment:
```suggestion
var AssumeRoleType = types.ObjectType{AttrTypes: map[string]attr.Type{
		"policy_arns":         types.SetType{ElemType: types.StringType},
		"transitive_tag_keys": types.SetType{ElemType: types.StringType},
		"tags":                types.MapType{ElemType: types.StringType},
		"duration":            types.StringType,
		"external_id":         types.StringType,
		"policy":              types.StringType,
		"role_arn":            types.StringType,
		"session_name":        types.StringType,
		"source_identity":     types.StringType,
	}
}
```
Can be directly defined as variable and used in ListValueFrom method

Reply:
Thank you!